Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -742,6 +771,18 @@ - (void)_showPhishingAlert
     [self presentViewController:alert.get() animated:YES completion:nil];
 }
 
+- (_WKExtrinsicButton *)_createButtonWithExtrinsicContentSize:(CGSize)size
+{
+    _WKExtrinsicButton *button = [_WKExtrinsicButton buttonWithType:UIButtonTypeSystem];
+    [button setTranslatesAutoresizingMaskIntoConstraints:NO];
+ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+    [button setAdjustsImageWhenHighlighted:NO];
```

Comment:
is there no non-deprecated API/SPI that can do this?

Reply:
We'd need to update all these buttons to use `UIButtonConfiguration`, which is outside of the scope of this PR.